frontend/src/components/maps/leaflet/InventoryLayer.tsx

```diff
@@ -246,16 +251,19 @@ export const InventoryLayer: React.FC<InventoryLayerProps> = ({
   }, [params, selected]);
 
   return (
-    <PointClusterer
-      points={features}
-      draftPoints={draftProperties}
-      zoom={zoom}
-      bounds={bbox}
-      onMarkerClick={onMarkerClick}
-      zoomToBoundsOnClick={true}
-      spiderfyOnMaxZoom={true}
-      selected={selected}
-      tilesLoaded={!loadingTiles}
-    />
+    <>
+      <PointClusterer
+        points={features}
+        draftPoints={draftProperties}
+        zoom={zoom}
+        bounds={bbox}
+        onMarkerClick={onMarkerClick}
+        zoomToBoundsOnClick={true}
+        spiderfyOnMaxZoom={true}
+        selected={selected}
+        tilesLoaded={!loadingTiles}
+      />
+      <FilterBackdrop show={showFilterBackdrop} />
```

Comment:
Why not always include the <FilterBackdrop/>, but only display it if set to true?

Reply:
Does not really matter, but this just makes is easy for reusability